config/environments/test.rb

```diff
@@ -15,6 +15,9 @@
   config.active_support.test_order = :random
   config.active_support.deprecation = :stderr
 
+  config.sendgrid_api_host = ENV['SENDGRID_API_HOST'] || 'https://sendgrid.example.com/'
+  config.sendgrid_api_user = ENV['SMTP_USERNAME']
+  config.sendgrid_api_key = ENV['SMTP_PASSWORD']
```

Comment:
I'll move as my initial assumption to need different ways of initialisation was wrong